### DISCUSSION TITLE
Functionality Request: Add "Album" to Information Screen

### DISCUSSION THREAD

**Discussion by Apesbrain** [LMS-Community/slimserver#564] [Ideas]
Would you consider adding to LMS the "Album" > (M) "More" functionality illustrated and described below?  Please forgive the formatting; this originated as a request of Material Skin.  Thanks.

![Clipboard01](https://user-images.githubusercontent.com/26585705/113717790-fc3e7580-96b9-11eb-8cf5-4643fec6cbed.jpg)

The "Album" listing would be bold, clickable and lead to this page:

![Clipboard03](https://user-images.githubusercontent.com/26585705/113727961-a4a50780-96c3-11eb-832f-b850b3b64acc.jpg)

NOTE: This functionality is somewhat implied in the first image above, but the first three links below the artwork do not work.

> The list that's produced by "More" comes from LMS, so it's up to LMS to add a link to the album there. I think you should create an issue on LMS's github for this.
>> Would you consider adding a feature for classical music?  Let's say you are listening to something by Beethoven and you want to hear more by him: right-click on Expanded Now Playing artwork and select "Go to composer".  You see a list of albums containing compositions by Beethoven and pick one you'd like to hear, but here's the thing: you'd like to hear the **entire** album not just the Beethoven composition(s) on that album.

>>Could you add a (...) > "More" > "Album" menu item with controls similar to "Album Artist", i.e. "Play now", "Append to queue"?  (See https://imgur.com/umTsI4N)  Thanks.

**Comment by michaelherger** [comment]
Wouldn't clicking one of the composers give you what you're looking for?

**Comment by Apesbrain** [comment]
Hi Michael, if you click on one of the Composers you only get his/her contribution(s) to that album.  What I'm suggesting is a way to play the **entire** album.  Thanks.
